etc/profile-m-z/qt6ct.profile

```diff
@@ -0,0 +1,69 @@
+# Firejail profile for qt6ct
+# Description: Qt6 Configuration Utility
+# This file is overwritten after every install/update
+# Persistent local customizations
+include qt6ct.local
+# Persistent global definitions
+ignore dns
+include globals.local
+
+blacklist /usr/libexec
+
+noblacklist ${HOME}/.config/qt6ct
+noblacklist ${HOME}/.local/share/qt6ct
```

Comment:
Not blacklisted.

Reply:
Sorry, I meant add the paths to disable-programs.inc rather than remove.

Reply:
These lines should be restored.

Reply:
> These lines should be restored.

Without removing the `blacklist` ones, as both are needed.

Anyway, I fixed and force-pushed.